export_img.py

```diff
@@ -0,0 +1,15 @@
+import sys
```

Comment:
Na zacatku by bylo fajn videt, k cemu ten soubor je a jak ho pouzit. Nyni to neni jasne, v kodu nejsou komentare :(